code/modules/power/apc.dm

```diff
@@ -814,7 +814,7 @@
 				else
 					to_chat(H, "<span class='warning'>You can't receive charge from the APC!</span>")
 			return
-		if((H.a_intent == INTENT_GRAB) && (E.drain_time < world.time))
+		else if(E.drain_time < world.time)
```

Comment:
This could maybe work better with combat mode right click instead?
Else ethereals will have a hard time regularly accessing APCs.

Reply:
ah yeah i mustve missed this, thanks ill check it.